### PR TITLE
[APP] Change embeding to product rating

### PR DESCRIPTION
**Pull request by zhoonit**
- [APP] Change embeding to product rating

```
This patch changes embedding application to product rating application
which is more practical use-case

**Changes proposed in this PR:**
- Update Readme
- Change directory and app name embedding->productratings
- tune model and sample dataset

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

### CODE REVIEW

**Comment by zhoonit** [comment]
@kparichay changing embedding application done, please build on top of this PR 👍

File: Applications/ProductRatings/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Product Ratings
+
+This application contains simple embeding based model that predicts ratings given user and product.
```
Comment (kparichay) [inline Applications/ProductRatings/README.md:3]:
```suggestion
This application contains a simple embedding-based model that predicts ratings given a user and a product.
```

File: Applications/ProductRatings/res/product_ratings.txt [hunk 2]
```diff
@@ -0,0 +1,25 @@
+1 1 5
```
Comment (kparichay) [inline Applications/ProductRatings/res/product_ratings.txt:1]: rename this file to `sample_product_ratings.txt`?

File: Applications/ProductRatings/res/product_ratings_model.ini [hunk 3]
```diff
@@ -16,36 +16,44 @@ epsilon = 1e-7
...
-split_dimension = 2 # split by height
+split_dimension = 3 # split by width
 
-[embedding1]
+[user_embed]
 input_layers = split
 Type = embedding
-in_dim = 15
-out_dim = 8
+in_dim = 6
```
Comment (kparichay) [inline Applications/ProductRatings/res/product_ratings_model.ini:28]: add a comment that this must be the max value of the `user id value + 1`.

File: Applications/ProductRatings/res/product_ratings_model.ini [hunk 4]
```diff
@@ -16,36 +16,44 @@ epsilon = 1e-7
...
+out_dim = 5
 in_length = 1
 
-[embedding2]
+[product_embed]
 input_layers = split
 Type = embedding
-in_dim = 15
-out_dim = 8
+in_dim = 6
```
Comment (kparichay) [inline Applications/ProductRatings/res/product_ratings_model.ini:35]: add a comment that this must be the max value of the `product id value + 1`.

**Review by kparichay — APPROVED** [review]
LGTM other than the minor comments.

File: Applications/ProductRatings/README.md (hunk 1 above)
Comment (zhoonit) [reply Applications/ProductRatings/README.md:3]: Done

File: Applications/ProductRatings/res/product_ratings.txt (hunk 2 above)
Comment (zhoonit) [reply Applications/ProductRatings/res/product_ratings.txt]: Done

File: Applications/ProductRatings/res/product_ratings_model.ini (hunk 3 above)
Comment (zhoonit) [reply Applications/ProductRatings/res/product_ratings_model.ini]: Done

File: Applications/ProductRatings/res/product_ratings_model.ini (hunk 4 above)
Comment (zhoonit) [reply Applications/ProductRatings/res/product_ratings_model.ini]: Done

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**